products/keycloak.md

```diff
@@ -0,0 +1,107 @@
+---
+title: Keycloak
+category: server-app
+# https://github.com/simple-icons/simple-icons/issues/8125
+iconSlug: NA
+permalink: /keycloak
+releasePolicyLink: https://github.com/keycloak/keycloak/security/policy#supported-versions
+changelogTemplate: "https://github.com/keycloak/keycloak/releases/tag/__LATEST__"
+activeSupportColumn: false
+releaseColumn: true
+releaseDateColumn: true
+eolColumn: Supported
+
+auto:
+-   git: https://github.com/keycloak/keycloak.git
+
+releases:
+-   releaseCycle: "20.0"
+    releaseDate: 2022-11-01
+    eol: false
+    latest: "20.0.2"
+    latestReleaseDate: 2022-11-13
+
+-   releaseCycle: "19.0"
+    releaseDate: 2022-07-27
+    eol: 2022-11-01
+    latest: "19.0.3"
+    latestReleaseDate: 2022-10-06
+
+-   releaseCycle: "18.0"
+    releaseDate: 2022-04-20
+    eol: 2022-10-06
+    latest: "18.0.2"
+    latestReleaseDate: 2022-06-24
+
+-   releaseCycle: "17.0"
+    releaseDate: 2022-03-11
+    eol: 2022-06-24
+    latest: "17.0.1"
+    latestReleaseDate: 2022-03-23
+
+-   releaseCycle: "16.1"
+    releaseDate: 2021-12-20
+    eol: 2022-03-23
+    latest: "16.1.1"
+    latestReleaseDate: 2022-01-25
+
+-   releaseCycle: "16.0"
+    releaseDate: 2021-12-17
+    eol: 2022-01-25
+    latest: "16.0.0"
+    latestReleaseDate: 2021-12-17
+
+-   releaseCycle: "15.1"
+    releaseDate: 2021-12-10
+    eol: 2021-12-17
+    latest: "15.1.1"
+    latestReleaseDate: 2021-12-17
+
+-   releaseCycle: "15.0"
+    releaseDate: 2021-07-15
+    eol: 2021-12-17
+    latest: "15.0.2"
+    latestReleaseDate: 2021-08-20
+
+-   releaseCycle: "14.0"
+    releaseDate: 2021-06-18
+    eol: 2021-08-20
+    latest: "14.0.0"
+    latestReleaseDate: 2021-06-18
+
+-   releaseCycle: "13.0"
+    releaseDate: 2021-05-06
+    eol: 2021-06-18
+    latest: "13.0.1"
+    latestReleaseDate: 2021-05-25
+
+-   releaseCycle: "12.0"
+    releaseDate: 2020-12-16
+    eol: 2021-05-25
+    latest: "12.0.4"
+    latestReleaseDate: 2021-03-01
+
+-   releaseCycle: "11.0"
+    releaseDate: 2020-07-22
+    eol: 2021-03-01
+    latest: "11.0.3"
+    latestReleaseDate: 2020-11-05
+
+-   releaseCycle: "10.0"
+    releaseDate: 2020-04-29
+    eol: 2020-11-05
+    latest: "10.0.2"
+    latestReleaseDate: 2020-06-02
+
+---
+
+> [Keycloak](https://www.keycloak.org/) is an open source identity and access management solution
+> for modern applications and services.
+
+Only the latest release is supported. Depending on the severity of a vulnerability the issue may be
+fixed in the current `major.minor` release or, for lower severity vulnerabilities or hardening, in
```

Comment:
this one looks almost same when you read so i suggest to emphasize  not just major.minor but also current and following parts too , OR we can just write it just like major +1  . minor for following version part

Reply:
Got this sentence on https://github.com/keycloak/keycloak/security/policy#supported-versions.

:-1:  for `major+1.minor`: the following version can also be a minor version.

I am more in favor of not changing anything. I agree this sentence is difficult to read but I did not found a better way to say it.

Reply:
my suggestion is : 
fixed in `the current major.minor` release or, for lower severity vulnerabilities or hardening, in `the following major.minor` release.

[ i mean taking `the current` and `the following` parts also inside  `` ]

Reply:
@marcwrobel

Reply:
That's strange, I do not think "current" and "following" must be formatted as code. I still vote for keeping the sentence like it is.

Reply:
or at least we completely remove both major.minor parts in sentences , because it makes them focused by reader and missing the important part ( `the current` and `the following` )

Reply:
Ok, let's do that.

Reply:
Done.